episodes/imaging-software.md

```diff
@@ -0,0 +1,486 @@
+you combine it with some Python scripting (although this isn't required for many 
+workflows!)
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+For this lesson, we will use Napari as our software of choice. It's worth 
+bearing in mind though that Fiji/ImageJ can be a useful alternative - and many 
+workflows will actually use both Fiji/ImageJ and Napari together! Again, it's 
+about choosing the right tool for your data and research question.
+
+
+## Opening Napari
+
+Let's get started by opening a new Napari window - you should have already 
+followed the [installation instructions](../learners/setup.md). Note this can 
+take a while the first time, so give it a few minutes!
+
+```bash
+conda activate napari-env
+napari
+```
+
+![](fig/blank-napari-ui.png){alt="A screenshot of the default Napari user 
+interface"}
+
+## Opening images
+
+Napari comes with some example images, let's open one now:
+
+```
+Go to the top menu-bar of Napari and select:
+File > Open Sample > napari builtins > Cells (3D+2Ch)
+```
+
+You should see a fluorescence microscopy image of some cells:
+
+![](fig/cells-napari.png){alt="A screenshot of a flourescence microscopy image 
+of some cells in Napari"}
+
+## Napari's User interface
+
+Napari's user interface is split into a few main sections, as you can see in the 
+diagram below:
+
+![](fig/ui-sections-napari.png){alt="A screenshot of Napari with the main user 
+interface sections labelled"}
+
+Let's take a brief look at each of these sections - for full information see the 
+[Napari documentation](https://napari.org/stable/tutorials/fundamentals/viewer.html).
+
+## Main menu
+
+We already used the main menu in the last section to open a sample image. The 
+main menu contains various commands for opening images, changing preferences and 
+installing plugins (we'll see more of these options in later episodes).
+
+## Canvas
+
+The canvas is the main part of the Napari user interface. This is where we 
+display and interact with our images. 
+
+Try moving around the cells image with the following commands:
+```
+Pan - Click and drag
+Zoom - Scroll in/out
+```
+
+## Dimension sliders
+
+Dimension sliders appear at the bottom of the canvas depending on the type of 
+image displayed. For example, if we drag the slider at the bottom of the cells 
+image, we move up and down in this 3D image stack.
+
+![](fig/dim-slider.png){alt="Three screenshots of the cells image in napari, at 
+different z depths"}
+
+Pressing the arrow buttons at either end of the slider steps through one image 
+at a time. Also, pressing the 'play' button at the very left of the slider moves 
+automatically through the stack until pressed again.
+
+We will see in later episodes that more sliders can appear if our image has more 
+dimensions (e.g. time series, or further channels).
+
+## Viewer buttons
+
+The viewer buttons control various aspects of the Napari viewer:
```

Comment:
I think it would help to crop out the viewer buttons from a previous image and annoate each of them so that people remember what you are talking about.

Reply:
I'll add a note that these are the buttons at the bottom left of Napari. As we now link to the icon svgs directly from the Napari repo, I think it's best to avoid also labelling them on a screenshot too ( so the image + icons don't get out of sync if the icons change )